portfolio/src/main/webapp/style.css

```diff
@@ -122,4 +125,66 @@ html, body {
 
 .tabText h1 {
     text-align: center;
+}
+
+/* GALLERY PROPERTIES */
+
+#galleryWrapper {
+    margin: 20px;
+    width: calc(100% - 40px);
+    overflow-y: auto;
+    max-height: calc(100% - 80px);
+}
+
+#galleryGrid {
+    display: grid;
+    grid-template-columns: 33% 33% 33%;
+}
+
+.gridItem {
+    height: 500px;
+    margin: 10px;
+    background-size: cover;
+    background-position-y: center;
+    box-shadow: 10px 10px 10px -8px;
+    border-radius: 20px;
+    display: flex;
+    align-items: flex-end;
+}
+
+@keyframes fadeIn {
```

Comment:
Animations are more commonly used when you have more than a start and end state, like if you had in-between states. A simpler way to do animations with only start and end states is to use CSS transitions https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Transitions/Using_CSS_transitions

Again, no solution is right or wrong. Just wanted to share a different tool.

Reply:
I agree, transitions would be much cleaner to use in this context.